src/invoker.js

```diff
@@ -22,11 +23,28 @@ var toString = require('./toString');
  * @return {Function} A new curried function.
  * @see R.construct
  * @example
+ *      // A function with no arguments
+ *      const asJson = invoker(0, "json")
+ *      // Just like calling .then((response) => response.json())
+ *      fetch("http://example.com/index.json").then(asJson)
+ *        
  *
- *      var sliceFrom = R.invoker(1, 'slice');
+ *      // A function with one arguments
+ *      var sliceFrom = invoker(1, 'slice');
  *      sliceFrom(6, 'abcdefghijklm'); //=> 'ghijklm'
- *      var sliceFrom6 = R.invoker(2, 'slice')(6);
+ *      
+ *      // A function with two arguments
+ *      var sliceFrom6 = invoker(2, 'slice')(6);
  *      sliceFrom6(8, 'abcdefghijklm'); //=> 'gh'
+ *
+ *      // NOTE: You can't simply pass some of the arguments to the initial invoker function.
+ *      const firstCreditCardSection = invoker(2, "slice", 0, 4)
+ *      firstCreditCardSection("4242 4242 4242 4242") // => Function<...>
+ *
+ *      // Instead you must immediately invoke the invoker (better wording tbd)
```

Comment:
How about:

"Since `invoker` returns a curried function, you may partially apply it to create the function you need."